mne/time_frequency/psd.py

```diff
@@ -112,16 +115,118 @@ def _check_nfft(n, n_fft, n_overlap):
+        Only use tapers with more than 90% spectral concentration within
+        bandwidth.
+    mt_normalization : str
+        Either "full" or "length" (default). If "full", the PSD will
+        be normalized by the sampling rate as well as the length of
+        the signal (as in nitime).
+    n_jobs : int
+        Number of CPUs to use in the computation.
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+
+    Returns
+    -------
+    psds : ndarray of the same input shape, final dimension is n_freqs
+        The power spectral densities.
+    freqs : ndarray (n_freqs)
+        The frequencies.
+    """
+    if data.ndim < 2:
+        raise ValueError('data must have >= 2 dimensions')
+    # Data prep
+    dshape = data.shape[:-1]
+    n_times = data.shape[-1]
+    data = data.reshape(np.product(dshape), n_times)
+    if method == 'welch':
+        n_fft, n_overlap = _check_nfft(n_times, wel_n_fft, wel_n_overlap)
+        win_size = n_fft / float(sfreq)
+        logger.info("Effective window size : %0.3f (s)" % win_size)
+
+        freqs = np.arange(n_fft // 2 + 1, dtype=float) * (sfreq / n_fft)
+        freq_mask = (freqs >= fmin) & (freqs <= fmax)
+        freqs = freqs[freq_mask]
+        psds = np.empty(data.shape[:-1] + (freqs.size,))
+        parallel, my_pwelch, n_jobs = parallel_func(_pwelch, n_jobs=n_jobs,
+                                                    verbose=verbose)
+        data_splits = np.array_split(data, n_jobs)
+        f_epochs = parallel(my_pwelch(d, noverlap=n_overlap, nfft=n_fft,
+                                      fs=sfreq, freq_mask=freq_mask,
+                                      welch_fun=welch)
+                            for d in data_splits)
+        psds = np.concatenate(f_epochs, axis=0)
+    elif method == 'multitaper':
+        # Stack data so it's treated separately
+        psds, freqs = multitaper_psd(x=data, sfreq=sfreq, fmin=fmin, fmax=fmax,
+                                     bandwidth=mt_bandwidth,
+                                     adaptive=mt_adaptive,
+                                     low_bias=mt_low_bias,
+                                     n_jobs=n_jobs,
+                                     normalization=mt_normalization,
+                                     verbose=verbose)
+    else:
+        raise ValueError("Unsupported PSD method: {0}".format(method))
+
+    n_freqs = len(freqs)
+    dshape = list(dshape) + [n_freqs]
+    psds = psds.reshape(dshape)
+    return psds, freqs
+
+
 @verbose
-def compute_epochs_psd(epochs, picks=None, fmin=0, fmax=np.inf, tmin=None,
-                       tmax=None, n_fft=256, n_overlap=0, proj=False,
-                       n_jobs=1, verbose=None):
+def compute_epochs_psd(epochs, picks=None, fmin=0, fmax=np.inf,
+                       tmin=None, tmax=None, proj=False, method='welch',
+                       wel_n_fft=256, wel_n_overlap=0, mt_bandwidth=None,
```

Comment:
arfff ... you can't do this. There needs to be a deprecation cycle etc. Why did you do this?